tools/osx/crosstool/cc_toolchain_config.bzl

```diff
@@ -5451,6 +5451,17 @@ def _impl(ctx):
         requires = [feature_set(features = ["opt"])],
     )
 
+    oso_prefix_feature = feature(
+        name = "oso_prefix_is_pwd",
+        flag_sets = [
+            flag_set(
+                actions = all_link_actions +
+                          ["objc-executable", "objc++-executable"],
```

Comment:
will this flag apply to executables without any objc as well?

Reply:
Should https://github.com/keith/bazel/blob/ks/add-oso_prefix_is_pwd-macos-feature/tools/osx/crosstool/cc_toolchain_config.bzl#L246-L250